tutorial/chapters/ch4_basic_batchapp/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+  "batch": {
+    "processes": [
+      {
+        "name": "bhptrainmodelnb",
+        "algorithmName": "main",
+        "namespace": "",
+        "algorithmTitle": {
+          "pt-br": "BHP Train Model NB",
+          "en-US": "BHP Train Model NB"
+        },
+        "algorithmDescription": {
+          "pt-br": "BHP Train Model NB",
+          "en-US": "BHP Train Model NB"
+        },
+        "instanceProperties": {
+          "profile": "",
+          "properties": {
+            "dockerImage": "bhptrainmodelnb/bhptrainmodelnb:1.0.0",
```

Comment:
"dockerImage" is deprecated no? it should be "dockerName", using dockerName we dont need to add the app name.. also no need for tag.. i.e.m 
`"dockerImage": "bhptrainmodelnb",`

Reply:
Changed both on config and doc figures.